psscriptanalyzer/Deploy-DemoApp.ps1

```
@@ -0,0 +1,46 @@
+    [string]$Environment = "production",
+    [string]$ArtifactPath
+)
+
+$Global:DeployConfig = @{
+    Environment = $Environment
+    Region      = "us-east-1"
+    StartedAt   = Get-Date
+}
+
+function Process-Database {
+    Write-Host "Migrating database for $($Global:DeployConfig.Environment)"
```

PSScriptAnalyzer (1.25.0)

[warning] Found global variable 'Global:DeployConfig'.

(PSAvoidGlobalVars)